src/engine-scripts/puppet/pinnableElementState.js

```diff
@@ -20,9 +20,11 @@ module.exports = async ( page, hashtags, featureName ) => {
 			const buttonSelector = `#vector-${featureName} .vector-pinnable-header-toggle-button`;
 			await page.evaluate( ( selector ) => {
 				const btn = document.querySelector( selector );
+				// Possible for the TOC or page tools to not be present on certain pages
 				if ( btn ) {
-					// Possible for the TOC to not be present
 					btn.click();
+					// Reset focus to reduce flakiness
+					document.activeElement = null;
```

Comment:
Thank you, I was noticing this too!

Reply:
Actually, I'm still noticing some flakiness with this:

<img width="1734" alt="Screen Shot 2023-03-13 at 10 54 19 AM" src="https://user-images.githubusercontent.com/3496279/224771392-15439ed1-95b9-45ef-8f30-19d2b3a92b7d.png">

I'm not sure document.activeElement can act as a setter? Perhaps, document.activeElement.blur() might work better

Reply:
ok good idea, i pushed that up!